openbas-front/src/admin/components/scenarios/scenario/ScenarioContext.ts

```diff
@@ -39,6 +39,11 @@ const injectContextForScenario = (scenario: ScenarioStore) => {
         resolve(response.data);
       }));
     },
+    onDryImportInjectFromXls(importId: string, input: InjectsImportInput): Promise<ImportTestSummary> {
+      return dryImportXls(scenario.scenario_id, importId, input).then((response) => new Promise((resolve, _reject) => {
+        resolve(response.data);
+      }));
+    },
     onBulkDeleteInjects(injectIds: string[]): void {
```

Comment:
I think you could just write: dryImportXls(scenario.scenario_id, importId, input)**.then**(response => response.data); (without wrapper into another promise)